### DISCUSSION TITLE
SymbolDisplayFormat for open generic types?

### DISCUSSION THREAD

**Discussion by alrz** [dotnet/roslyn#74584] [Q&A]
There is `SymbolDisplayGenericsOptions.IncludeTypeParameters` but it omits the parameter list entirely. 

Is there a option to print open generic types for `typeof`?

**Comment by 333fred** [comment]
I don't believe we have such an API today.

**Reply by alrz** [reply]
Also the formatter for fully qualified metadata name `QualifiedNameArityFormat` is internal which uses `SymbolDisplayCompilerInternalOptions.UseArityForGenericTypes`.

 Current workaround for both is to use `FullyQualifiedFormat` and adjust afterwards. Should I convert this to an API proposal?

**Reply by 333fred** [reply]
Sure, we can look at that.
